tests/models/superpoint/test_modeling_superpoint.py

```diff
@@ -0,0 +1,274 @@
+import inspect
+import unittest
+from typing import List
+
+from transformers.models.superpoint.configuration_superpoint import SuperPointConfig
+from transformers.testing_utils import require_torch, require_vision, slow, torch_device
+from transformers.utils import cached_property, is_torch_available, is_vision_available
+
+from ...test_configuration_common import ConfigTester
+from ...test_modeling_common import ModelTesterMixin, floats_tensor
+
+
+if is_torch_available():
+    import torch
+
+    from transformers import (
+        SUPERPOINT_PRETRAINED_MODEL_ARCHIVE_LIST,
+        AutoModelForInterestPointDescription,
+        SuperPointModel,
+        SuperPointModelForInterestPointDescription,
```

Comment:
`SuperPointModelForInterestPointDescription` has to be renamed to `SuperPointForInterestPointDescription`

Reply:
Fixed in [this commit](https://github.com/huggingface/transformers/pull/25786/commits/a8f7e1c97b89d563552c0e32c350eb7ebead1506)